src/views/home/components/deposit-limit-modal/deposit-limit-modal.view.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC } from "react";
+import { SOFT_DEPOSIT_LIMIT_EXCEEDED_WORD } from "src/constants";
+
+import { FormData } from "src/domain";
+import { useDepositLimitModalStyles } from "src/views/home/components/deposit-limit-modal/deposit-limit-modal.styles";
+import { TextMatchForm } from "src/views/home/components/text-match-form/text-match-form.view";
+import { Button } from "src/views/shared/button/button.view";
+import { Card } from "src/views/shared/card/card.view";
+import { Portal } from "src/views/shared/portal/portal.view";
+
+export type DepositLimitModalData =
```

Comment:
I'd name this just `Limit` or `DepositLimit` and would use the same name for the `data` prop below.

Reply:
I've already created a `DepositLimit` interface in the domain:

```ts
interface DepositLimit {
  hard: number;
  soft: number;
}
```

Take into account that this interface refers to the Modal being shown `warning` or `forbidden`, I think that the current name really describes what it is.